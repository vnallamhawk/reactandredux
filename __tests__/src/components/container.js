import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import Container from '../../../container';

describe('Container Test -- Snapshot',()=>{
    it('capturing snapshot of container',()=>{
        const renderer = new ShallowRenderer();
        const result = renderer.render(<Container/>)
        expect(result).toMatchSnapshot();
    })
})