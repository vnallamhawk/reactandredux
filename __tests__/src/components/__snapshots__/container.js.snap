// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container Test -- Snapshot capturing snapshot of container 1`] = `
<div>
  <Connect(Retail) />
</div>
`;
